### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by vieiramanda11**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi Amanda, this is a very good start, you nailed it. Just one small thing left, the div tag holding those images are actually links, wrap them in an `<a>` tag make sure they are clickable, and submit for re-review.  Kudos again for a great start

Also, you should check out Github flow, and start using it from the next project see [this](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/)

**Comment by vieiramanda11** [comment]
Hello Raphael!
Thank you for the feedback and the tips.
I did the required changes.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                <li><a href=#>Mac</a></li>
+                <li><a href=#>iPod</a></li>
+                <li><a href=#>iPhone</a></li>
+                <li><a href=#>iPad</a></li>
+                <li><a href=#>iTunes</a></li>
+                <li><a href=#>Support</a></li>
+                <li><i class="fa fa-search"></i><input type="text"></li>
+            </ul>
+        </nav>
+        <div class="title"></div>
```
Comment (Oghenebrume50) [inline index.html]: These divs are also links too, just correct that, merge to master and Submit the project completion form, Good Job.

**Review by Oghenebrume50 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <div class="subheading"></div>
+        <div class="subheading-2"></div>
```
Comment (Oghenebrume50) [inline index.html]: These divs are also links too, just correct that, merge to master and Submit the project completion form, Good Job.

**Review by Oghenebrume50 — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (vieiramanda11) [reply index.html]: Done, Raphael! Thank you again.

**Pull request merged**